cmd/ssh-auth-none-demo/ssh-auth-none-demo.go

```diff
@@ -1,187 +0,0 @@
-// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
Is this necessary/used?

Reply:
Yes, please keep it. We run it on the internet and point SSH client app developers at it to fix their code without requiring them learn/use Tailscale.

Reply:
Does it need to behave the same way? I ask because with the current golang.org/x/crypto, we won't be able to send all of the banners that we used to.

Reply:
I'm really souring on unforking if we have to keep making these compromises.

I think we should delay unforking until we upstream the API changes we need.